src/main/java/apka/config.json

```diff
@@ -0,0 +1 @@
+{"gpsPort":1911,"gpsIP":"0.0.0.0","gprsIP":"0.0.0.1","port":7779,"ip":"0.0.0.1","gprsPort":9111}
```

Comment:
tylko, że to musi być jednolite, dlatego wg. mnie rozwiązaniem jest po prostu tablica/lista obiektów które trzymają te parametry. Przez to będziesz mógł się przeiterować po nich i zrobić wszystko co trzeba